blues/application/tasks.py

```diff
@@ -224,24 +224,53 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository().get('url', '')
+    if url or not url.startswith('git@github.com:'):
+        return None
+
+    domain, repo = url.split(':')
+    org, repo = repo.split('/')
```

Comment:
I'm not sure about that, because that would make that function a rather GitHub specific thing, hmm..